### DISCUSSION TITLE
Allow Overriding Column Name In Collection Page

### DISCUSSION THREAD

**Discussion by zicklag** [directus/directus#15510] [Archive]
Hey there!

I have a simple scenario where I have articles and tags with a M2M relationship between them. In my collection page for articles, I want to display the slugs of the tags related to that article, but when I use the slug of the tag, the column name in the collection page shows `Slug`, which makes it look like the column is the _article's_ slug, when it fact it is the tags slug. I would like to clear up this confusing labeling by re-naming the column.

Here's a short screen-cast to show what I'm talking about. Logically, the name of the column I'm adding should be "Tags" or "Tag Slugs", but I can't change it from "Slug".

https://user-images.githubusercontent.com/25393315/189445484-d5426d32-e6d6-4077-8e29-9e7740f91c42.mp4

**Comment by azrikahar** [comment]
Feel free to upvote #13277 and/or #12269 as that is where the bulk of the discussion on this topic resides in! 👍 Marking this as archived so that we can converge further inputs/thoughts in the linked discussions.
